bin/check-marathon-task.rb

```diff
@@ -30,13 +30,13 @@
 #
 
 require 'sensu-plugin/check/cli'
-require 'net/http'
+require 'rest-client'
```

Comment:
what is the value of using `rest-client` here? You can set headers with `net/http`: http://ruby-doc.org/stdlib-2.4.2/libdoc/net/http/rdoc/Net/HTTPHeader.html

Reply:
I see it was done for consistency sake. :+1:

Reply:
yes, I think it's because all the other checks are using `rest-client`